.github/workflows/nextjs-integration-test.yml

```diff
@@ -129,6 +129,7 @@ jobs:
           corepack disable
           pnpm install
           pnpm run build
+          echo "RUNNING NEXTJS VERSION: $(packages/next/dist/bin/next --version)"
```

Comment:
Probably worth a comment that this is a marker that the action above depends on, and it shouldn't be changed.

Reply:
Yup, updated.